extensions/binding/org.eclipse.smarthome.binding.hue/src/main/java/org/eclipse/smarthome/binding/hue/internal/discovery/BridgeJsonParameters.java

```diff
@@ -0,0 +1,49 @@
+/**
+ * Copyright (c) 2014-2016 by the respective copyright holders.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.eclipse.smarthome.binding.hue.internal.discovery;
+
+/**
+ * The {@link BridgeJsonParameters} class defines JSON object, which
+ * contains bridge attributes like IP address. It is used for bridge
+ * N-UPNP Discovery.
+ *
+ * @author Awelkiyar Wehabrebi
+ *
+ */
+public class BridgeJsonParameters {
+
+    // hue bridge parameters
+    private final String id;
+    private final String internalipaddress;
```

Comment:
Makes sense. You could use 

````java 
@SerializedName("internalipaddress") private final internalIpAddress
```` 
since the json returned does not using camel case. Similar on all other places.